pbxproj/pbxextensions/ProjectFlags.py

```diff
@@ -203,21 +210,24 @@ def remove_framework_search_paths(self, paths, target_name=None, configuration_n
         :param configuration_name: Configuration name to add the flag to or None for every configuration
         :return: void
         """
-        self.remove_search_paths(XCBuildConfigurationFlags.FRAMEWORK_SEARCH_PATHS, paths, target_name, configuration_name)
+        self.remove_search_paths(
+            XCBuildConfigurationFlags.FRAMEWORK_SEARCH_PATHS, paths, target_name, configuration_name)
 
-    def add_run_script(self, script, target_name=None, insert_before_compile=False, input_files=None, output_files=None):
+    def add_run_script(self, script, target_name=None, insert_before_compile=False, input_files=None, output_files=None, run_install_build=0):
         """
         Adds a run script phase into the given target, optionally before compilation phase
         :param script: Script to be inserted on the run script
         :param target_name: Target name or list of target names to add the run script to or None for every target
         :param insert_before_compile: Insert the run script phase before the compilation of the source files. By default,
         it's added at the end.
         :param input_files: An array of input file paths to be added to the run script
-        :param output_files: An array of output file paths to be added to the run script
```

Comment:
The documentation for the `output_files` doc is removed and `input_files` is duplicated